app/questionnaire/placeholder_transforms.py

```diff
@@ -41,10 +47,39 @@ def __init__(
 
     input_date_format = "%Y-%m-%d"
 
-    def format_currency(self, number: Decimal | float, currency: str = "GBP") -> str:
-        formatted_currency: str = format_currency(number, currency, locale=self.locale)
+    def format_currency(
+        self,
+        number: int | Decimal | float,
+        raw_args: Mapping,
+        currency: str = "GBP",
+    ) -> str:
+        """
+        The raw arguments for the transform are required here, in addition to the formatted number, as custom logic is required
+        to calculate the correct number of decimals based on the source of the transform.
```

Comment:
It should mention, the decimal only takes into account if source is answer or calc summary without any previous transform.

We need to also capture that in the decision logs.

Reply:
Added to the decision log now 👍